NDB.Covid19/NDB.Covid19.Droid/Utils/DialogUtils.cs

```diff
@@ -48,7 +48,7 @@ public static void DisplayBubbleDialog(Activity current, string message, string
                 return tcs.Task;
             }
 
-            var dialog = new AlertDialog.Builder(activity, Android.Resource.Style.ThemeDeviceDefaultLightDialog)
```

Comment:
@nc-kano was there a reason why it was set to light mode?

Reply:
Light theme is default in the app, so light/dark mode is not changed here. But default theme is material alert, while this manual theme setting used the android alert instead. (dont know if this is why it was manually set though).

Reply:
@pavshr As i remember we were observing issues where on some devices it was not readable or not matching the style of the app (probably some Huawei phones with forced dark mode but i may be wrong here)